.env.example

```
@@ -1,14 +1,22 @@
 DATABASE_URL="postgres://postgres:postgres@localhost:5432/fsrsDemo?schema=public&pool_timeout=0"
 
 
-NEXTAUTH_URL=http://localhost:3000
-NEXTAUTH_SECRET=lh5H/LzVhuP98GJjYsvtObebM/BpkANcANANKIW9TtI=
+# Better Auth
+BETTER_AUTH_URL=http://localhost:3000
 # openssl rand -base64 32
+BETTER_AUTH_SECRET=lh5H/LzVhuP98GJjYsvtObebM/BpkANcANANKIW9TtI=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The BETTER_AUTH_SECRET key should go before the BETTER_AUTH_URL key

(UnorderedKey)